changelog.d/8744.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug where appservices may be sent an excessive amount of read receipts and presence.
```

Comment:
I think you may want to add the same change to `_handle_receipts`?

Reply:
We do not. We are happy to give a stream key of 0 because we fetch the last 100 events whatever happens, so it will fetch the last 100, then store the latest token, then move on with it's life.